tests/e2e/support/commands.js

```diff
@@ -21,6 +21,13 @@ const transformGameUrl = (api, slug) => {
     return Cypress.Promise.resolve(`/api/${api}/${slug}`);
   }
 
+  if (slug === 'rematch') {
+    return cy
+      .window()
+      .its('cuttle.gameStore.id')
+      .then((gameId) => `/api/game/${gameId}/rematch`);
+  }
```

Comment:
This will get tricky if we ever need to call rematch on behalf of users who are in a game from the perspective of the 'main' user who is _not_ in the game e.g. being on the home page while other people rematch since the game id isn't available in the store at that time. For the moment this is fine and we'll cross that bridge if and when we come to it.